modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/Subscriptions/index.js

```diff
@@ -4,16 +4,35 @@ import {getAccountSubscriptions} from '../../../../common/services/liferay/graph
 import CardSubscription from '../CardSubscription';
 import SubscriptionsNavbar from '../SubscriptionsNavbar';
 
+export const POSSIBLE_STATUS_AMOUNT = 3;
+
 const Subscriptions = ({accountKey}) => {
 	const [selectedSubscriptionGroup, setSelectedSubscriptionGroup] = useState(
-		''
+		() => ''
 	);
-	const [selectedStatus, setSelectedStatus] = useState('');
-
+	const [selectedStatus, setSelectedStatus] = useState(() => [
+		'Active',
+		'Expired',
+		'Future',
+	]);
+	
 	const parseAccountSubscriptionGroupERC = (subscriptionName) => {
 		return subscriptionName.toLowerCase().replace(' ', '-');
 	};
 
+	const mountQueryString = (acc, cv, index, array) => {
```

Comment:
No abbreviations for parameters;

```suggestion
	const getAccountSubscriptionFilterQueryString = (previousValue, currentValue, currentIndex, array) => {
```

Reply:
Ok, updating.